sequelize/migrations/20201103113401-create-player.js

```diff
@@ -15,6 +15,9 @@ module.exports = {
                 allowNull: false,
                 type: DataTypes.STRING
             },
+            avatar: {
+                type: DataTypes.STRING
+            },
```

Comment:
Why have you edited the old migration file, if you want to modify a database schema create a new migration for that

Reply:
Done